x-pack/plugins/enterprise_search/public/applications/shared/tables/inline_editable_table/inline_editable_table.tsx

```diff
@@ -124,9 +123,11 @@ export const InlineEditableTableContents = <Item extends ItemWithAnID>({
     <>
       <EuiFlexGroup alignItems="center">
         <EuiFlexItem>
-          <EuiTitle size="xs">
-            <h3>{title}</h3>
-          </EuiTitle>
+          {!!title && (
+            <EuiTitle size="xs" data-test-subj="title">
```

Comment:
Nit: Can we use something a bit more descriptive than "title" here? the data-test-subj attr is for testing and this would help make tests more readable if this is ever used in a test.

Reply:
Good point, I was basing this on the other data-test-subj attr names in the file, but looking around the Kibana codebase they tend to be more specific. I updated the attr names.